tests/components/homewizard/test_sensor.py

```diff
@@ -310,6 +308,20 @@
                 "sensor.device_wi_fi_ssid",
             ],
         ),
+        (
+            "HWE-P1-unavailable-exports",
```

Comment:
This test proofs the fix of https://github.com/home-assistant/core/issues/133379, where a missing `energy_export_tariff_1` trigger a removal of `energy_import_tariff_1`